src/commands/ipmi.rb

```diff
@@ -85,16 +82,12 @@ def node_names
         @node_names ||= nodes.map(&:name)
       end
 
-      def libvirt_info
-        object = options.group ? group : node
-        {
-          host: object.answer.libvirt_host,
-          vm: object.answer.is_vm,
-        }
+      def libvirt_host(node)
+        node.config.libvirt_host
```

Comment:
Don't think you need this method here any more, as the `libvirt_host` is now only used within `Vm`.